patches/html-to-image+1.10.8.patch

```diff
@@ -1,25 +1,5 @@
 diff --git a/node_modules/html-to-image/es/embed-images.js b/node_modules/html-to-image/es/embed-images.js
 index 70e8468..e04eb19 100644
---- a/node_modules/html-to-image/es/embed-images.js
-+++ b/node_modules/html-to-image/es/embed-images.js
-@@ -21,7 +21,6 @@ async function embedImageNode(clonedNode, options) {
-         : clonedNode.href.baseVal;
-     const dataURL = await resourceToDataURL(url, getMimeType(url), options);
-     await new Promise((resolve, reject) => {
--        clonedNode.onload = resolve;
-         clonedNode.onerror = reject;
-         if (clonedNode instanceof HTMLImageElement) {
-             clonedNode.srcset = '';
-@@ -30,6 +29,7 @@ async function embedImageNode(clonedNode, options) {
-         else {
-             clonedNode.href.baseVal = dataURL;
-         }
-+        clonedNode.decode().then(() => resolve(clonedNode)).catch(reject);
```

Comment:
This was added within #12623 but it seems to break the logic for FF and Safari. Looks like removing this doesn't influence the Karma tests (the same tests are still failing as on `main`), so not sure exactly why this was added for the tests.

Reply:
It's pretty confusing to read a diff of a diff 😂